apps/functions/applications-migration/package.json

```diff
@@ -23,7 +23,7 @@
     "mysql2": "^3.9.8",
     "pins-data-model": "github:Planning-Inspectorate/data-model#1.4.1",
     "sequelize": "^6.33.0",
-    "tedious": "^18.2.0"
+    "tedious": "^11.0.8"
```

Comment:
Is it intentional we are downgrading?

Reply:
yes, looks like `tedious >=11.0.9` uses an old version of `azure-identity` (3.4.1) so to fix a vulnerability with it it downgrades it. Will look into it, Pr set to `DRAFT` for now

Reply:
I discussed this with @JamesP-KC and `tedious` is an important dependency to access ODW and downgrading it to such an earlier version will most likely carry breaking changes.

The vulnerability that `azure-identity <4.2.1` has is classed as `medium` and seems low risk for auth code that isn't public facing.

There is an issue raised on the `tedious` repo about this, so hopefully a new version will not be too far away, which uses the new version of azure-identity: https://github.com/tediousjs/tedious/issues/1633.

So the solution for now would be to upgrade all the instances of `azure-identity` to 4.2.1 on our `package.json` files but leave `tedious` to 18 even if it depends on `azure-identity 3.4.1`.

@beejjacobs, @JamesP-KC , @Yakitrak does this sound good?

Reply:
That sounds like a good idea to me

Reply:
Yup sounds sensible.